Sources/Datadog/RUM/RUMVitals/VitalRefreshRateReader.swift

```diff
@@ -53,25 +54,47 @@ internal class VitalRefreshRateReader: ContinuousVitalReader {
         }
     }
 
-    // MARK: - Private
+    // MARK: - Internal
```

Comment:
nit/ Any reason to change following methods from `private` to `internal`? With `private` we were enforcing thread safety for the entire reader. Now it becomes conventional - all methods must be called on the same thread.

If this is for testability of `framesPerSecond(provider:)` then it seems a good compromise 👍. Even better would be to isolate FPS logic into separate unit that would be private to the reader, but internal for its own unit tests.

It's more a question on the reasoning, than change request 🙂.

Reply:
I was kind of expecting. It is just me being lazy and didn't want to move code around a lot. But I agree, Frame detector should be its own unit.

Reply:
I just moved back to private and kept `framesPerSecond(provider:)` internal for tests. I think it is good compromise.

Reply:
Sounds good 👍 as we're touching existing code here (vs adding new code).